### PR TITLE
Nested Clauses

### PR DESCRIPTION
**Pull request by gxb5443**
Nests Clauses to implement a Polish Notation implementation of clauses for complex queries.

### CODE REVIEW

File: arnoldb.proto [hunk 1]
```diff
@@ -77,10 +77,36 @@ message Objects {
...
+    }
+
+    message Branch {
+      LogicalOperator lop = 1;
+      Clause left = 2;
+      Clause right= 3;
+    }
+
+    enum LogicalOperator {
+      NOOP = 0;
```
Comment (poopoothegorilla) [inline arnoldb.proto]: What is an example of when NOOP will be used?

**Comment by poopoothegorilla** [comment]
related to https://github.com/namely/arnoldb/pull/96/files

File: arnoldb.proto (hunk 1 above)
Comment (gxb5443) [inline arnoldb.proto]: Not filling it in.

**Comment by poopoothegorilla** [comment]
+1

**Pull request merged**